digraph "unitGraph" {
    "r5 = staticinvoke <javaT.lang.reflect.Array.ExceedMaxDim: java.lang.Object getArrayOf256Dimensions()>()"
    "$r0 = virtualinvoke r5.<java.lang.Object: java.lang.Class getClass()>()"
    "staticinvoke <java.lang.reflect.Array: java.lang.Object newInstance(java.lang.Class,int)>($r0, 1)"
    "goto [?= $r1 = new java.lang.Exception]"
    "$r2 := @caughtexception"
    "r3 = $r2"
    "$r4 = <java.lang.System: java.io.PrintStream out>"
    "virtualinvoke $r4.<java.io.PrintStream: void println(java.lang.String)>(\"success: newInstanceOne test\")"
    "return"
    "$r1 = new java.lang.Exception"
    "specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"NewArray allowed dimensions > MAXDIM\")"
    "throw $r1"
    "r5 = staticinvoke <javaT.lang.reflect.Array.ExceedMaxDim: java.lang.Object getArrayOf256Dimensions()>()"->"$r0 = virtualinvoke r5.<java.lang.Object: java.lang.Class getClass()>()";
    "r5 = staticinvoke <javaT.lang.reflect.Array.ExceedMaxDim: java.lang.Object getArrayOf256Dimensions()>()"->"$r2 := @caughtexception";
    "$r0 = virtualinvoke r5.<java.lang.Object: java.lang.Class getClass()>()"->"staticinvoke <java.lang.reflect.Array: java.lang.Object newInstance(java.lang.Class,int)>($r0, 1)";
    "$r0 = virtualinvoke r5.<java.lang.Object: java.lang.Class getClass()>()"->"$r2 := @caughtexception";
    "staticinvoke <java.lang.reflect.Array: java.lang.Object newInstance(java.lang.Class,int)>($r0, 1)"->"goto [?= $r1 = new java.lang.Exception]";
    "staticinvoke <java.lang.reflect.Array: java.lang.Object newInstance(java.lang.Class,int)>($r0, 1)"->"$r2 := @caughtexception";
    "goto [?= $r1 = new java.lang.Exception]"->"$r1 = new java.lang.Exception";
    "$r2 := @caughtexception"->"r3 = $r2";
    "r3 = $r2"->"$r4 = <java.lang.System: java.io.PrintStream out>";
    "$r4 = <java.lang.System: java.io.PrintStream out>"->"virtualinvoke $r4.<java.io.PrintStream: void println(java.lang.String)>(\"success: newInstanceOne test\")";
    "virtualinvoke $r4.<java.io.PrintStream: void println(java.lang.String)>(\"success: newInstanceOne test\")"->"return";
    "$r1 = new java.lang.Exception"->"specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"NewArray allowed dimensions > MAXDIM\")";
    "specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"NewArray allowed dimensions > MAXDIM\")"->"throw $r1";
}
